docs/user/kind-bridge-ipv4-config.yaml

```diff
@@ -0,0 +1,41 @@
+# this config file contains all config fields with comments
```

Comment:
if this file is added should it be linked from the docs?
possibly a separate PR should add this file and link to docs?

Reply:
Perhaps an "examples" folder would be more appropriate. We will likely have additional configurations for different scenarios -- e.g. IPv4/IPv6 single/multi node with various cni plugins.